blocks/divider/README.md

```diff
@@ -1,5 +1,12 @@
-The Divider block is an `<hr>` element which can be customized with [`https://blockprotocol.org/@blockprotocol/types/property-type/height-in-pixels/`](https://blockprotocol.org/@blockprotocol/types/property-type/height-in-pixels/) and [`https://blockprotocol.org/@blockprotocol/types/property-type/css-background-color/`](https://blockprotocol.org/@blockprotocol/types/property-type/css-background-color/) properties.
+The Divider block is a customizable wrapper around an `<hr>` element.
 
-Its width is set to 100% and its border to "none", allowing the [`https://blockprotocol.org/@blockprotocol/types/property-type/height-in-pixels/`](https://blockprotocol.org/@blockprotocol/types/property-type/height-in-pixels/) property to control the line's thickness. The [`https://blockprotocol.org/@blockprotocol/types/property-type/css-background-color/`](https://blockprotocol.org/@blockprotocol/types/property-type/css-background-color/) property sets the element's background color.
+## Programmatic Usage
```

Comment:
I'm unsure what the hub is going to do with these headings, if it does go wrong, it feels like something we should be able to deal with.

Reply:
The new Figma designs for the Þ Hub specifically show how we want to handled these styled headers, so I suggest we leave them in for now (since the current experience isn't _dire_).

Reply:
What I meant is I'm not sure how the README rendering component is going to do when it sees high-level headings (like `h1` or `h2`).

Unless we patch them, or override their styling to be smaller, it might be quite strange

Reply:
I'm sure it won't be too hard to fix, and should hopefully be relatively quick to hotfix if needs be, just saying the hub _might_ look kinda terrible until we patch it (if there's a problem), and we should check when we publish the blocks